bootstrap.sh

```diff
@@ -1,27 +1,46 @@
 #!/usr/bin/env bash
 # Usage: ./bootstrap.sh
-# This script will install swiftformat, link git hooks and install required ruby gems.
+# This script will install Mint and bootstrap its dependencies, link git hooks and install required ruby gems.
 # You should have homebrew installed.
 # If you get `zsh: permission denied: ./bootstrap.sh` error, please run `chmod +x bootstrap.sh` first
 
-# Install swiftformat 0.45.6
-brew unlink swiftformat
-brew tap-new getstream/ios
-brew extract --version 0.45.6 swiftformat getstream/ios
-brew reinstall getstream/ios/swiftformat@0.45.6
-brew untap getstream/ios
+SWIFTFORMAT_VERSION=0.45.6
+SWIFTLINT_VERSION=0.41.0 # Also used by danger-swiftlint via bundle config
+
+# Check if Homebrew is installed
+if [[ $(command -v brew) == "" ]]; then
+    echo "Homebrew not installed. Please install."
+    exit 1
+fi
+
+echo
+echo "👉 Generate mintfile"
+echo "# Warning - this file is generated using bootstrap.sh, don't edit it directly." > mintfile
+echo "nicklockwood/SwiftFormat@$SWIFTFORMAT_VERSION" >> mintfile
+echo "realm/SwiftLint@$SWIFTLINT_VERSION" >> mintfile
+cat mintfile
+
+echo
+echo -e "👉 Install Mint if needed"
+# List installed Mint versions, if fails, install Mint
+brew list mint || brew install mint
```

Comment:
IMHO This seems too complicated :D 

Let me explain. Right now, we are doing: 

1. Install one package manager (🍺).
1. This 🍺 doesn't really install anything unless step 3.
1. With 🍺 we do install now 🌱 .
1. with 🌱 we finally do our nice thing and install `Swiftlint` and `Swiftformat`

Personally I would get rid of the dependencies and install this stuff just in the scope of the repo, also I secretely hate 🍺 because it can ruin your environment if misused or misunderstood which happens quite often.

1. Swiftlint can be installed via `CocoaPods`. Or Just 🍺. Both are already used before this PR.
2. SwiftFormat can be stand-alone command line tool (without any use of any package manager...) Or it can be installed via brew, downsideof the stand-alone thing is we would need to update it everytime it gets updated... 

My idea is to either:

- Use 🍺 for both Swiftlint and Swiftformat
- Use `cocoaPods` for `Swiftlint` (nicely integrates into the project), Use `standalone binary` of `swiftformat`
- (Your idea)

Reply:
`Homebrew` is bad, because:
1. You cannot lock version, it always install most recent one
2. You cannot install old one except via dirty tricks (look at previous installation of SwiftFormat)
3. Install in global environment

We don't want to use `CocoaPods`, because:
1. We are not using them right now
2. super heavy and invasive into your project

We don't want use global installations:
1. I have multiple projects with different versions of SwiftLint, with global installation no way for me to handle it
2. We need to be able sync tools versions through team (writing in slack is not a way)

Reply:
@ManWithBear is right. I know it's not ideal to have 3 tools to install our dependencies but Homebrew doesn't provide us all we need. We use RubyGems and Bundler to run Fastlane, Mint to install all Swift-based dependencies, and Homebrew to install Mint :) Since almost everyone has Homebrew and it's also preinstalled on the CI, I think it's a setup we can live with.